db/migrate/20180521172343_create_offers.rb

```diff
@@ -0,0 +1,14 @@
+class CreateOffers < ActiveRecord::Migration[5.1]
+  def change
+    create_table :offers do |t|
+      t.references :product
+      t.text :description
```

Comment:
If you add validations in model add them in migration as well. So you need to find out how to do it :)

Reply:
Well, I have to keep looking for how to do that :)

Reply:
https://stackoverflow.com/questions/13122791/rails-validation-in-model-vs-migration?utm_medium=organic&utm_source=google_rich_qa&utm_campaign=google_rich_qa